packages/graphql/schema.graphql

```diff
@@ -0,0 +1,205 @@
+### This file was generated by Nexus Schema
+### Do not make changes to this file directly
+
+
+input AddProjectInput {
+  isCurrent: Boolean!
+  projectRoot: String!
+  testingType: String!
+}
+
+"""Namespace for information related to the app"""
+type App {
+  """The active project in the app"""
+  activeProject: Project
+
+  """Whether this is the first open of the application or not"""
+  isFirstOpen: Boolean!
+
+  """All known projects for the app"""
+  projects: [Project!]!
+}
+
+"""
+A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the `date-time` format outlined in section 5.6 of the RFC 3339 profile of the ISO 8601 standard for representation of dates and times using the Gregorian calendar.
+"""
+scalar DateTime
+
+enum FrontendFramework {
+  cra
+  nextjs
+  nuxtjs
+  react
+  vue
+  vuecli
+}
+
+"""
+The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
+"""
+scalar JSON @specifiedBy(url: "http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf")
+
+type Mutation {
+  """Adds a new project to the app"""
+  addProject(input: AddProjectInput!): Project!
+
+  """Initializes the plugins for the current active project"""
+  initializePlugins: Project
+
+  """Installs the dependencies for the component testing step"""
+  wizardInstallDependencies: Wizard
+
+  """Navigates backward in the wizard"""
+  wizardNavigateBack: Wizard
+
+  """Navigates forward in the wizard"""
+  wizardNavigateForward: Wizard
```

Comment:
We can probably simplify this to `wizardNavigate(direction: forward | backward)`

Reply:
Sure, let's get it merged and figure out the quality of life improvements - they'll be a lot easier to review in their own context.

Reply:
Done here: https://github.com/cypress-io/cypress/pull/17551/files